### DISCUSSION TITLE
Page Feature - doesn't look like it working

### DISCUSSION THREAD

**Discussion by psfletcher** [Lissy93/dashy#961] [Help and Support]
Hi All,
I've followed the how to's on the website and have added the pages function to my conf.yml file
As per one of the discussion on github i've tried path: 'networking.yml' and path: networking.yml with no luck.
pages:
- name: Networking Services
  path: networking.yml
- name: Work Stuff
  path: work.yml

I do get "Network Services" and "Work Stuff" buttons on the navlink bar next to my static links.
But when i press them i get
http://<WebServer>/home/networking-services - but the page doesn't update
And then when i refresh the page i get  - Cannot GET /home/networking-services as the body of the page.

Any ideas what's going on please?

**Comment by KenK73** [comment]
Looks like this is already being tracked as a bug:

** #[BUG] Multi-Page Support , can´t get it to work. #1005**

It's Jan 2023 and I am having the same issue.
